Add Promscale install using package cloud and refactor install layout (#1228)
timescale/docs: promscale/installation/debian-ubuntu.md

import PromscaleConnector from 'versionContent/_partials/_promscale-connector.mdx';

# Install Promscale on Debian or Ubuntu
You can install Promscale on any Debian or Ubuntu system where you have an
already installed and working Prometheus or OpenTelemetry environment.

## Install TimescaleDB and the Promscale extension
To use Promscale you need a TimescaleDB database, as well as the Promscale
extension. The Promscale extension contains support functions to improve
the performance of Promscale.

<procedure>

### Installing TimescaleDB and Promscale extension

1.  Install TimescaleDB following the instructions in the
    [TimescaleDB install page][tsdb-install-self-hosted]
1.  Install the Promscale extension:
    ```
    apt install promscale-extension-postgresql-14
    ```
    
</procedure>

## Install the Promscale connector

> <PromscaleConnector >

Reviewer: ```suggestion
<PromscaleConnector />
```

import PromscaleConnector from 'versionContent/_partials/_promscale-connector.mdx';

# Install Promscale on Debian or Ubuntu
You can install Promscale on any Debian or Ubuntu system where you have an
already installed and working Prometheus or OpenTelemetry environment.

## Install TimescaleDB and the Promscale extension
To use Promscale you need a TimescaleDB database, as well as the Promscale
extension. The Promscale extension contains support functions to improve
the performance of Promscale.

<procedure>

### Installing TimescaleDB and Promscale extension

1.  Install TimescaleDB following the instructions in the
    [TimescaleDB install page][tsdb-install-self-hosted]
1.  Install the Promscale extension:
    ```
    apt install promscale-extension-postgresql-14
    ```
    
</procedure>

## Install the Promscale connector

> <PromscaleConnector />

Reviewer: ```suggestion

<PromscaleConnector />
```

install the Promscale connector using this command:

```bash
apt install promscale

> 

Reviewer 2: ```

import PromscaleConnector from 'versionContent/_partials/_promscale-connector.mdx';

# Install Promscale on Debian or Ubuntu
You can install Promscale on any Debian or Ubuntu system where you have an
already installed and working Prometheus or OpenTelemetry environment.

## Install TimescaleDB and the Promscale extension
To use Promscale you need a TimescaleDB database, as well as the Promscale
extension. The Promscale extension contains support functions to improve
the performance of Promscale.

<procedure>

### Installing TimescaleDB and Promscale extension

1.  Install TimescaleDB following the instructions in the
    [TimescaleDB install page][tsdb-install-self-hosted]
1.  Install the Promscale extension:
    ```

>     apt install promscale-extension-postgresql-14

Reviewer 3: @VineethReddy02 I think we also provide packages for pg 12 and 13 right? Let's change this so it's clear.

I guess we could add a note saying something like replace 14 with the corresponding PostgreSQL major version. Versions 12, 13 and 14 are supported.

Or we change the command to apt install promscale-extension-postgresql-<pg_version> and explain they have to replace <pg_version> with 12, 13 or 14. I would use the former since I expect most people installing from scratch would be using 14and it reduces a little bit the amount of work the user has to do (they can just copy and paste)

Author: For the time being, I will add a note below this cmd. In the future, I will create a page for the compatibility matrix.

    ```
    
</procedure>

## Install the Promscale connector

<PromscaleConnector />

    ```

>        apt install promscale

Reviewer 3: This instructions will not work if the user installs the Connector in a different host because we not telling anywhere they have to set up the package repos. We'll have to add those instructions here somehow.

> # Install Promscale on Debain or Ubuntu

Reviewer: ```suggestion
# Install Promscale on Debian or Ubuntu
```

You can install Promscale on any Debian or Ubuntu system where you have an
already installed and working Prometheus or OpenTelemetry environment.

> ## Install TimescaleDB and Promscale extension

Reviewer: ```suggestion
## Install TimescaleDB and the Promscale extension
```

> To use with Promscale you need TimescaleDB database, as well as the Promscale

Reviewer: ```suggestion
To use Promscale you need a TimescaleDB database, as well as the Promscale
```

extension. The Promscale extension contains support functions to improve

> performance of Promscale.

Reviewer: ```suggestion
the performance of Promscale.
```

<procedure>

### Installing TimescaleDB and Promscale extension

1.  Install TimescaleDB following the instructions in the
    [TimescaleDB install page][tsdb-install-self-hosted]

> 1.  Install Promscale extension

Reviewer: ```suggestion
1.  Install the Promscale extension:
```

    ```
    apt install promscale-extension-postgresql-14
    ```
    
</procedure>

> 

Reviewer: ```suggestion
```

## Install the Promscale connector

> The Promscale connector understands PromQL queries natively and fetches data

Reviewer: ```suggestion
The Promscale connector consumes PromQL queries natively and fetches data
```

?? Not sure which verb to use here, but "understands" feels a little too much like anthropomorphism 😉

from TimescaleDB to execute them, while SQL queries go directly to TimescaleDB.

> After you install TimescaleDB and Promscale extension, install Promscale
> connector.
> 
> <procedure>
> 
> 1.  Install Promscale connector:
>     ```bash
>     apt install promscale
>     ```
> 
> </procedure>

Reviewer: ```suggestion
After you install TimescaleDB and the Promscale extension, you can 
install the Promscale connector using this command:

```bash
apt install promscale
```

```

Reviewer: The backticks have broken the suggestion, but you get the idea, I'm sure 🤦‍♀️
